amundsen_application/api/utils/notification_utils.py

```diff
@@ -192,13 +192,14 @@ def _log_send_notification(*, notification_type: str, options: Dict, recipients:
         )
 
         response = mail_client.send_email(
-            recipients=recipients,
-            sender=sender,
             subject=subject,
+            text='Notification text',  # TODO generate a proper text (now html being used instead anyway)
             html=html,
             optional_data={
                 'email_type': notification_type,
             },
+            sender=sender,
```

Comment:
nit: Might was well put the others in alphabetical order if that was the intent :)

Reply:
No, the reason was to rearrange parameters to place required ones first and then optional ones. But yes, I can place each group in alphabetical order as well 👍